source/v2/openapi.yaml

```diff
@@ -1313,6 +1313,31 @@ paths:
           source: |-
             curl -X GET https://www.trade-tariff.service.gov.uk/api/v2/rules_of_origin_schemes/392113/FR
 
+  /preference_codes:
+    get:
+      summary: Retrieves a list of all the preference codes
+      description: |
+        Retrieves a list of preference codes
```

Comment:
Do we want to explain what a preference code is or link out to an explanation of a preference code?

Reply:
I've managed to find this chipolata: https://www.revenue.ie/en/customs/documents/electronic/aep-trader-guide-appendix-17.pdf

Reply:
👍 good point, Matt decided to go with: https://www.gov.uk/government/publications/preference-codes-for-data-element-417-of-the-customs-declaration-service